pkg/plugins/manager/signature/manifest.go

```diff
@@ -314,3 +309,66 @@ func urlMatch(specs []string, target string, signatureType plugins.SignatureType
 	}
 	return false, nil
 }
+
+type invalidFieldErr struct {
+	field string
+}
+
+func (r invalidFieldErr) Error() string {
+	return fmt.Sprintf("valid manifest field %s is required", r.field)
+}
+
+func validateManifest(m pluginManifest, block *clearsign.Block) error {
+	if len(m.Plugin) == 0 {
+		return invalidFieldErr{field: "plugin"}
+	}
+	if len(m.Version) == 0 || !isValidSemVer(m.Version) {
```

Comment:
Are we sure that our current plugins comply with being a valid semver version?

Reply:
Ha good point. I'll remove both version checks 👍